partials/forms/select.html

```diff
@@ -1,12 +1,26 @@
-<div id="{{id}}-group" class="form-group{{#compound}} form-group-compound{{/compound}}{{#error}} error{{/error}}">
+<div id="{{id}}-group" class="form-group{{#compound}} form-group-compound{{/compound}}
+{{#error}} error{{/error}}
+{{#groupClassName}}
+    {{groupClassName}}
+{{/groupClassName}}"
```

Comment:
It seems like the pattern is already broken. Should we fix them now? It would mean a breaking change?